db/migrate/20230730145516_create_schools.rb

```diff
@@ -2,6 +2,7 @@ class CreateSchools < ActiveRecord::Migration[7.0]
   def change
     create_table :schools do |t|
       t.text :name, null: false
+      t.integer :academic_year_id, null: false
```

Comment:
We can't alter a previously run migration, because it's already been played. Once migration files are merged to the main branch of a repository, and then (assumed) run against the production database, they become dead code files. The only way to add the `academic_year_id` to `School` is to create a new migration that only adds a column.